README.md

```diff
@@ -76,7 +76,7 @@ The 7.0.0 release changes the required boilerplate in two aspects.
           case message of
             ...
             Mdl message' ->
-              Material.update message' model
+              Material.update update message' model
```

Comment:
Or this?  Looking at the `Material.update` function it seems to not be the case that it changed this way...